addons/dark-www/addon.json

```
@@ -2124,7 +2124,7 @@
   "latestUpdate": {
     "version": "1.34.0",
     "newSettings": ["messageIndicatorColor", "messageIndicatorOnMessagesPage"],
-    "isMajor": false
+    "temporaryNotice": "There are now presets for Scratch 2.0 and the colors used on the front page between 2015 and 2018. The message badge colors are also customizable now."
```

- If we're calling the preset `Scratch's default colors (light blue)`, there won't be an easy way to know which one was the one that was used by Scratch from 2015 to 2018.
- I'd say "message badge color" singular because both badges are similar, just with multiple variants.

```suggestion
    "temporaryNotice": "There are two new presets - one resembling Scratch 2.0 and one with a light blue accent color, which was used on the front page between 2015 and 2018. The message badge color is also customizable now."
```